lib/block_view.go

```diff
@@ -543,6 +546,13 @@ func (pe *PostEntry) IsDeleted() bool {
 	return pe.isDeleted
 }
 
+func IsQuotedReclout(postEntry *PostEntry) bool {
+	if postEntry.IsQuotedReclout && postEntry.RecloutedPostHash != nil {
```

Comment:
can't we just return `postEntry.IsQuotedReclout && postEntry.RecloutedPostHash != nil`?

Reply:
resolved with #16